Make/Test-develop.yml

```
@@ -1,258 +1,258 @@
+    environment:
+      - TZ=Americas/Los_Angeles
+    network_mode: bridge
+    ports:
+      - 7205:7001
+    volumes:
+      - test_wisenetwave-lsio_config:/config
+      - test_wisenetwave-lsio_media:/media
+      - test_wisenetwave-lsio_backup:/backup
+      - test_wisenetwave-lsio_analytics:/analytics
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (2 > 0)

(empty-lines)